content/chainguard/chainguard-enforce/policies/chainguard-policies-cli.md

```diff
@@ -138,6 +138,67 @@ chainctl policies update enforce-policy --description ""
 
 Essentially, this example command deletes the `enforce-policy` policy's description.
 
+## Policy versions
+
+Policy versions enable you to gradually test and roll out policy changes to your cluster over time, without losing the historical context of how a given policy has changed. Versions also increase your visibility into when and by whom policies were updated. With policy versions, you can quickly rollback to a stable version in the event of the inevitable typo or unintended consequence discovered too late.
+
+### Reviewing policy versions in chainctl
+
+‍No special action is needed to create a new version of a policy; any change to the policy document will automatically create a new version. Note that updates to a policy’s description will not create a new version. Differences between versions can be inspected through either `chainctl` or the Console.
+
+Listing policies with the `-owide` or `--output=wide` flag shows some new policy metadata, including created and updated timestamps, the currently enforced version of the policy, and the author of the enforced version.
+
+```sh
+chainctl policy ls -owide
```

Comment:
```suggestion
chainctl policy ls -o wide
```

This is just a personal readability preference, but I like when an argument is separated from the corresponding flag